vendor/engines/bulk_email/app/views/bulk_email/jobs/show.html.haml

```diff
@@ -0,0 +1,23 @@
+= content_for :tabnav do
+  = render "bulk_email/nav_tab", secondary_tab: "history"
+
+= content_for :h1 do
+  = current_facility
+
+%h2= t("bulk_email.title")
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_users", sidenav_tab: "bulk_email"
+
+= readonly_form_for :bulk_email_job do |f|
+  = f.input :sender
+  = f.input :created_at
+  = f.input :recipient_list
+  = f.input :subject
+  = f.input :body
```

Comment:
Should we `simple_format` this like we do the actual email?

I did discover that `simple_format` will allow some HTML through (like `<strong>`, `<a>`, and `<h1>` tags), but does sanitize for script. I think I'm ok with that. Or, according to the docs, we should invoke `h` before passing to simple_format (both here and the email itself).

http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html